chapter-10/end/Objects/Text/LivesText.cs

```diff
@@ -0,0 +1,27 @@
+﻿using chapter_10.Engine.Objects;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace chapter_10.Objects.Text
+{
+    public class LivesText : BaseTextObject
+    {
+        private int _nbLives = -1;
+
+        public int NbLives {
+            get
+            {
+                return _nbLives;
+            }
+            set
+            {
+                _nbLives = value;
+                Text = "Lives: " + _nbLives;
```

Comment:
Maybe use the newer string literal format:

`Text = $"Lives: {_nbLives}";`

Reply:
done!